MAUI/Maps/tooltip.md

```diff
@@ -445,8 +549,13 @@ Data template can be used customize the tooltip view using [`ShapeTooltipTemplat
 ![Maps tooltip appearance customization](images/tooltip/tooltip_textStyle.png)
 
 N>
-* The BindingContext of the Shape and Bubble will be the corresponding underline object.
-* The BindingContext of the Marker will be the corresponding MapMarker.
+* The BindingContext of the Tooltip will be the `TooltipInfo`, it holds the `DataItem` property. This property has corresponding underline object for shape, bubble and marker.
+
+N>
+* Refer to the `ShapeTooltipSettings` for customize the shape tooltip appearance.
+* Refer to the `BubbleTooltipSettings` for customize the bubble tooltip appearance.
+* Refer to the `MarkerTooltipSettings` for customize the marker tooltip appearance.
+
 
 N>
 * Refer to the [`ShapeTooltipTemplate`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html#Syncfusion_Maui_Maps_MapShapeLayer_ShapeTooltipTemplate), for customize tooltip for the shapes.
```

Comment:
* Refer to the [`ShapeTooltipTemplate`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html#Syncfusion_Maui_Maps_MapShapeLayer_ShapeTooltipTemplate) to customize the tooltip for the shapes.

Reply:
I have changed.